src/Symfony/Component/Form/Extension/Core/Type/DateType.php

```diff
@@ -321,7 +321,7 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('months', 'array');
         $resolver->setAllowedTypes('days', 'array');
         $resolver->setAllowedTypes('input_format', 'string');
-        $resolver->setAllowedTypes('calendar', ['null', \IntlCalendar::class]);
```

Comment:
I thought this was on purpose: just passing a "traditional" calendar (via the const) is useless, because which calendar? so that an IntlCalendar is required, because it would contain the complete description. Am I wrong?